cloud/services/bastionhosts/mocks_bastionhosts/client_mock.go

```diff
@@ -27,31 +27,31 @@ import (
 	reflect "reflect"
 )
 
-// MockClient is a mock of Client interface.
-type MockClient struct {
+// Mockclient is a mock of client interface.
+type Mockclient struct {
```

Comment:
this looks a little strange, I would have expected camel case... but I know it's auto-generated so nothing we can do